src/containers/shared/components/Transaction/NFTokenMint/Simple.tsx

```diff
@@ -4,12 +4,23 @@ import { useTranslation } from 'react-i18next'
 import { SimpleRow } from '../SimpleRow'
 import { TransactionSimpleComponent, TransactionSimpleProps } from '../types'
 import { NFTokenMintInstructions } from './types'
+import Account from '../../Account'
+import { useLanguage } from '../../../hooks'
+import { localizeNumber } from '../../../utils'
 
 export const Simple: TransactionSimpleComponent = ({
   data,
 }: TransactionSimpleProps<NFTokenMintInstructions>) => {
-  const { tokenID, tokenTaxon, uri } = data.instructions
+  const { tokenID, tokenTaxon, uri, transferFee, issuer } = data.instructions
   const { t } = useTranslation()
+  const language = useLanguage()
+  const formattedFee =
+    transferFee &&
+    `${localizeNumber((transferFee / 1000).toPrecision(5), language, {
```

Comment:
TransferFee fraction of the amount a NFToken is transferred for.  It shouldn't be treated as a currency.